containers/run_hydra.sh

```diff
@@ -199,6 +199,21 @@ if [ "$SRV" != "" ] ; then
     MOUNTS="$MOUNTS --mount type=bind,source=${SRV},target=/srv"
 fi
 
+KNOWN_HOSTS="${STORE}/home/.ssh/known_hosts"
+if [ "${HC_GEN_KNOWN_HOSTS,,}" = "yes" ] && [ ! -f "$KNOWN_HOSTS" ]; then
+    echo "Creating known_hosts"
+    mkdir -p "$(dirname "$KNOWN_HOSTS")"
+    touch "$KNOWN_HOSTS"
+    chmod og-rwx "$(dirname "$KNOWN_HOSTS")" "$KNOWN_HOSTS"
+    for host in "${HC_CUSTOM_HOSTS[@]}"; do
+        h_name=${host%%:*}
+        h_ip=${host##*:}
+        echo "Scanning host keys for ${h_name} (${h_ip})"
+        ssh-keyscan -H "$h_name" >> "$KNOWN_HOSTS"
+        ssh-keyscan -H "$h_ip" >> "$KNOWN_HOSTS"
```

Comment:
This should work for everything else except awsarm, as ssh-keyscan assumes ssh is using default port 22 and awsarm uses 20220.

For non-default port you would need to do something like:

`ssh-keyscan -H -p 20220 awsarm.vedenemo.dev`

Reply:
You're correct.
I suppose then it requires another list after all, with port numbers...

Reply:
Added a separate list for ssh-hosts.